java/hadoop/src/main/java/com/youtube/vitess/hadoop/VitessConf.java

```diff
@@ -97,4 +99,12 @@ public void setAlgorithm(SplitQueryRequest.Algorithm algorithm) {
   public void setRpcFactoryClass(Class<? extends RpcClientFactory> clz) {
     conf.set(RPC_FACTORY_CLASS, clz.getName());
   }
+
+  public Query.ExecuteOptions.IncludedFields getIncludedFields() {
+    return conf.getEnum(INCLUDED_FIELDS, Query.ExecuteOptions.IncludedFields.TYPE_AND_NAME);
+  }
+
+  public void setIncludedFields(Query.ExecuteOptions.IncludedFields includedFields) {
```

Comment:
@ashudeep-sharma in the other PR you commented on this.  You're right I need to add a call to it to the VitessInputFormat.setInput and MapreduceIT classes.  I will do that in a separate PR if you dont mind, to keep this simple.